mappings/net/minecraft/nbt/NbtList.mapping

```diff
@@ -1,7 +1,14 @@
-CLASS net/minecraft/class_2499 net/minecraft/nbt/ListTag
+CLASS net/minecraft/class_2499 net/minecraft/nbt/NbtList
+	COMMENT Represents an NBT list.
+	COMMENT <p>
+	COMMENT An NBT list holds values of the same {@linkplain NbtElement#getType NBT type}.
+	COMMENT The {@linkplain AbstractNbtList#getListType NBT type} of an NBT list is determined
+	COMMENT once its first element is inserted; empty NBT lists return {@code 0} as their held {@linkplain AbstractNbtList#getListType NBT type}.
```

Comment:
Maybe `AbstractNbtList#getListType` should be renamed to `getHeldType`, since this seems to be a better name unless anyone disagrees.

Reply:
Describing it as an element type rather than a held type would be more appropriate for a list

Reply:
That's where we started. I don't think it makes sense to name it `getElementType` since everything is an `NbtElement`.

Reply:
I feel like we are falling into the rabbit hole where `Element` `Entry` `Content` can all be ambiguous and cause problems